### ISSUE TITLE
False negative with calling a TypeVar

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#9497]
```py
from typing import TypeVar

T = TypeVar("T")
T() # runtime error: TypeError: 'TypeVar' object is not callable
```

**Comment by hauntsaninja** [comment]
Thanks, duplicate of #9244

**Comment by hauntsaninja** [comment]
Opened a PR with a fix! Thanks again :-)
